core/unit_test/TestExecSpacePartitioning.hpp

```diff
@@ -79,6 +79,11 @@ void check_distinctive([[maybe_unused]] ExecSpace exec1,
 #ifdef KOKKOS_ENABLE_OPENMP
 template <class Lambda1, class Lambda2>
 void run_threaded_test(const Lambda1 l1, const Lambda2 l2) {
+#ifdef KOKKOS_ENABLE_OPENMP
+  if (Kokkos::OpenMP().concurrency() == 1)
+    GTEST_SKIP() << "test needs at least two OpenMP threads";
+#endif
```

Comment:
```suggestion
  if (omp_get_max_threads() == 1)
    GTEST_SKIP() << "insufficient number of supported concurrent threads";
```
Same comment everywhere in this file and the thread safety one